pkg/review/config.go

```diff
@@ -27,9 +28,10 @@ import (
 // Config defines the set of environment variables required
 // for running the artifact job.
 type Config struct {
-	GitHubAppID            string `env:"GITHUB_APP_ID,required"`             // The GitHub App ID
-	GitHubInstallID        string `env:"GITHUB_INSTALL_ID,required"`         // The provisioned GitHub App Installation reference
-	GitHubPrivateKeySecret string `env:"GITHUB_PRIVATE_KEY_SECRET,required"` // The secret name & version containing the GitHub App private key
+	GitHubEnterpriseServerURL string `env:"GITHUB_ENTERPRISE_SERVER_URL"`       // The GitHub Enterprise Server instance URL, format "https://[hostname]"
+	GitHubAppID               string `env:"GITHUB_APP_ID,required"`             // The GitHub App ID
+	GitHubInstallID           string `env:"GITHUB_INSTALL_ID,required"`         // The provisioned GitHub App Installation reference
```

Comment:
This is going to need to be changed to look up the install id instead of having it passed in from the command line / environment. 

You'll have an installation ID per org and we can't have an instance of GMA per org. You'll need to get the installation id for each org/repo somehow like what we do in minty: https://github.com/abcxyz/github-token-minter/blob/main/pkg/server/source/github.go#L71

Reply:
Make sense! I will do it in a separate CL since it requires some refactor.